packages/node-core/src/indexer/fetch.service.spec.ts

```diff
@@ -284,25 +285,26 @@ describe('Fetch Service', () => {
 
   it('enqueues blocks WITH valid dictionary results', async () => {
     enableDictionary();
-
+    expect((fetchService as any).useDictionary).toBeTruthy();
     const enqueueBlocksSpy = jest.spyOn(blockDispatcher, 'enqueueBlocks');
     const dictionarySpy = jest.spyOn(dictionaryService, 'scopedDictionaryEntries');
-
     await fetchService.init(1);
-
-    expect((fetchService as any).useDictionary).toBeTruthy();
+    // wait enqueue completed
+    await delay(3);
```

Comment:
I don't think this is a good idea, it leads to unreliable tests. Its a good indication that there is something wrong with our code if we have to do this.